lib/selenium-webdriver/webdriver.js

```diff
@@ -25,20 +39,37 @@ goog.provide('webdriver');
 webdriver.WebDriver = function() {};
 
 /**
- * Creates a new action sequence using this driver. The sequence will not be
+ * Creates a sequence of mouse actions using this driver. The sequence will not be
```

Comment:
Can't this include other things like `sendKeys`? I don't think we should specify 'mouse' actions.